addons/binding/org.openhab.binding.lutron/ESH-INF/thing/keypads.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't have a clear picture of how to split them up, or even if they need to be split up, so for now I've combined them all into thing-types.xml.